perfkitbenchmarker/providers/gcp/gcp_managed_relational_db.py

```diff
@@ -149,8 +158,21 @@ def _IsReady(self):
     Returns:
       True if the resource was ready in time, False if the wait timed out.
     """
+    cmd = util.GcloudCommand(self, 'sql', 'instances', 'describe',
+                             self.instance_id)
+    stdout, _, _ = cmd.Issue()
+    json_output = json.loads(stdout)
```

Comment:
Either is fine, but gcloud does return 1 on describe if the instance does not exist, so that is why I wrote it that way. May be more fragile than looking at the json.